spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/PodUtils.java

```diff
@@ -27,13 +27,8 @@
 
 	/**
 	 * @return A supplier of the currentPod Pod. The supplier will hold the currentPod pod
-	 * if inside Kubernetes or false, otherwise.
+	 * if inside Kubernetes or null, otherwise.
 	 */
 	Supplier<T> currentPod();
 
-	/**
```

Comment:
I  personally doubt the usefulness of this method. If you want to find out if this is running in k8s or not, you call `currentPod` and check if the result is `!=null`, just like the javadoc says.